src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -76,25 +76,25 @@ def writeGenotypes(pedigree, genoProbFunc):
     args = InputOutput.args
     if not args.no_dosage:
         writeDosages(pedigree, genoProbFunc, args.out + ".dosage.txt")
-    if args.haps:
-        writeGenoProbs(pedigree, genoProbFunc, args.out + ".haps")
+    if args.phased_geno_prob:
+        writeGenoProbs(pedigree, genoProbFunc, args.out + ".phased_geno_prob.txt")
 
-    if args.calling_threshold is not None:
-        for thresh in args.calling_threshold:
+    if args.geno_threshold is not None:
+        for thresh in args.geno_threshold:
             if args.binary_call_files:
                 writeBinaryCalledGenotypes(
                     pedigree, genoProbFunc, args.out + ".called." + str(thresh), thresh
                 )
-            if not args.binary_call_files:
+            if args.geno:
```

Comment:
Please modify line 88 to `if not args.binary_call_files and args.geno:`

Reply:
Update.